src/poetry/core/utils/_compat.py

```diff
@@ -12,7 +12,7 @@
 try:
     FileNotFoundError
 except NameError:
-    FileNotFoundError = IOError  # noqa
+    FileNotFoundError = IOError
```

Comment:
ditto here - can we use `if sys.version_info > ...`?

Reply:
same as above

Reply:
ping @tony

Reply:
In #263 I pushed a change removing this